pom.xml

```diff
@@ -396,6 +371,116 @@
 	</build>
 
 	<profiles>
```

Comment:
I think profiles aren't the right move here because we don't really want users to have to do this kind of switching, etc. It would be better, I think to have two parents and have the SDK archetype pick one if we really can't get away from the hard-coded dependencies.

Reply:
Hey @ibacher 
i have just created to POM files with a base POM file to avoid redundancy by centralizing common settings
i am wondering am i required to setp also the archetype file and seting up thr conditional pom that is according to your comment 
please clarify more for me so that i can see the point